### DISCUSSION TITLE
Hyperion App

### DISCUSSION THREAD

**Discussion by ghfb2020** [awawa-dev/HyperHDR#549] [Q&A]
Is there a way to make the Hyperion app work?  While HyperHDR is way better, I did like the Hyperion app, and I can't get it to work with HyperHDR.  If there is no way to make it work, is there a way to put WLED on a set up running HyperSPI?

**Comment by awawa-dev** [comment]
Hi
No, there is no way. Currently, HyperHDR's schemes and API are different, and the old Android app to control it will not work properly. HyperHDR is optimized for use with mobile browsers by using the latest version of boostrap 5 and other supporting libraries, and most of the interface has been rewritten to be compatible with it. If you want a native app to control it, you need to develop a new app.
